src/agreements/CollateralAgreement.sol

```diff
@@ -124,11 +126,18 @@ contract CollateralAgreementFramework is IAgreementFramework {
         emit AgreementFinalizationSent(id, msg.sender);
     }
 
-    function disputeAgreement(uint256 id) public {}
+    function disputeAgreement(uint256 id) external override {
```

Comment:
Is this a dispute of a _joined_ agreement or a _finalized_ agreement? Add a quick explanation to NatSpec?

Reply:
Just realized that there are already comments in the `IAgreementFramework` interface, so ignore my previous comment 😆